package.json

```diff
@@ -1,80 +1,80 @@
-{
-  "name": "@adminjs/express",
-  "version": "4.0.0",
-  "description": "This is an official AdminJS plugin which integrates it with Express.js framework",
-  "main": "lib/index.js",
-  "scripts": {
-    "dev": "rm -rf lib && tsc --watch",
-    "build": "rm -rf lib && tsc",
-    "example:simple": "nodemon",
-    "example:auth": "nodemon --config nodemon.auth.json",
-    "test": "jest --config ./test/jest.json",
-    "lint": "eslint './**/*.ts'",
-    "check:all": "yarn lint && yarn build && yarn test",
-    "release": "semantic-release"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/SoftwareBrothers/adminjs-expressjs.git"
-  },
-  "keywords": [
-    "expressjs",
-    "admin",
-    "adminjs",
-    "admin-panel"
-  ],
-  "author": "Michał Laskowski",
-  "license": "SEE LICENSE IN LICENSE",
-  "bugs": {
-    "url": "https://github.com/SoftwareBrothers/adminjs-expressjs/issues"
-  },
-  "homepage": "https://github.com/SoftwareBrothers/adminjs-expressjs#readme",
-  "peerDependencies": {
-    "adminjs": ">=5.0.0",
-    "express": ">=4.16.4",
-    "express-formidable": "^1.2.0",
-    "express-session": ">=1.15.6"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
-    }
-  },
-  "devDependencies": {
-    "@adminjs/mongoose": "^2.0.0",
-    "@commitlint/config-conventional": "^11.0.0",
-    "@semantic-release/git": "^9.0.0",
-    "@types/express": "^4.17.9",
-    "@types/express-formidable": "^1.0.4",
-    "@types/express-serve-static-core": "^4.17.13",
-    "@types/express-session": "^1.17.3",
-    "@types/jest": "^26.0.15",
-    "@types/mongoose": "^5.10.1",
-    "@typescript-eslint/eslint-plugin": "^4.8.1",
-    "@typescript-eslint/parser": "^4.8.1",
-    "adminjs": "^5.0.0",
-    "chai": "^4.2.0",
-    "commitlint": "^11.0.0",
-    "eslint": "^7.13.0",
-    "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-prettier": "^3.1.4",
-    "express": "^4.17.1",
-    "express-formidable": "^1.2.0",
-    "express-session": "^1.17.1",
-    "husky": "^4.3.0",
-    "jest": "^26.6.3",
-    "mongoose": "^5.10.15",
-    "nodemon": "^2.0.6",
-    "prettier": "^2.2.0",
-    "semantic-release": "^17.2.4",
-    "semantic-release-jira-releases-sb": "^0.7.2",
-    "semantic-release-slack-bot": "^1.6.2",
-    "sinon": "^9.2.1",
-    "sinon-chai": "^3.5.0",
-    "ts-jest": "^26.4.4",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.1.2"
-  }
-}
+{

```

Comment:
Not sure what happened that it detects that whole file's contents changed? 🤔

Reply:
LF -> CRLF